src/zope/publisher/tests/test_browserlanguages.py

```diff
@@ -83,9 +88,10 @@ def test_setPreferredLanguages(self):
 
     def test_conflicting_adapters(self):
         request = TestRequest("da, en, pt")
-        not_compatible_browser_languages = BrowserLanguages(request)
+        BrowserLanguages(request)
```

Comment:
Is this line even doing anything useful?

Reply:
No, it is not needed at all.